src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -91,17 +93,20 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
 
     form.onExport(() -> new Thread(() -> {
       List<String> errors = export();
+      int exportedFormsAmount = forms.getSelectedForms().size();
```

Comment:
I gave up making the distinction between “less” and “fewer” (“10 items or less” at the market no longer bothers me), but I do make the distinction between number and amount. Is it the *number* of exported forms?

Reply:
Hmmm, I didn't know about that distinction... I'll research, thanks!